src/content/structured/accessibility/documenting/accessibility-statements-guidance.mdx

```diff
@@ -44,11 +44,11 @@ The [when to create and update](/accessibility/documenting/when-to-create-and-up
 
 ### How to use the template
 
-You may need to make minor changes such as changing between singular and plural or 'app to service' but don't re-order, change titles or remove anything unless this guidance tells you to.
+You may need to make minor changes, such as changing between singular and plural or 'app to service', but don't re-order, change titles or remove anything unless this guidance tells you to.
 
 Keep the language simple. This statement needs to be understandable for everyone in the department, it shouldn't be technical.
 
-Update everything in square brackets (between '['and ']') - remove the square brackets.
+Update everything in square brackets (between '[' and ']') – remove the square brackets.
```

Comment:
There are several instances across the website where a hyphen should be an en rule or an em rule. Is there a way we can show the different types? Like using &ndash; and &mdash; as suggested here:[ ](https://stackoverflow.com/questions/52371734/how-to-make-a-long-in-markdown-i-e-the-character)

Reply:
I'd give using the HTML entities a go. That will make the characters more distinct in the .mdx files and hopefully will render on the page properly (let me know if you need guidance on running the site locally to test it out!)